packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -46,135 +53,105 @@
             "$ref": "#/definitions/backgroundImage",
             "description": "Image to display in the background of this page. It should be a horizontal image and it will be fixed at the bottom of the page."
           },
-          "layout": {
```

Comment:
These json-schema diffs are a little hard to read, it's probably easier to view the resulting file directly (via the `...` button in github).